web/api/queries/query.c

```diff
@@ -440,6 +444,12 @@ static inline void do_dimension_variablestep(
                 // use the previously read database value
                 n_curr = n_prev;
             } else {
+                if (no_db_one_time == 1 && rd->rrd_memory_mode != RRD_MEMORY_MODE_DBENGINE)
+                {
+                    no_db_one_time = 0;
+                    rd->state->query_ops.next_metric(&handle, &db_now);
+
```

Comment:
Please, remove this unnecessary blank line.
 I observed that sometimes you wrote `if () {`, and sometimes you start the `{` in the other line. The majority of our code starts the `{` in the same line of the `if`, I suggest you do the same for we keep a pattern.